source/wrangling.md

```diff
@@ -1110,32 +1132,26 @@ has index `1`!).
 tidy_lang.iloc[:, 1]
 ```
 
-You can also ask for multiple columns,
-we pass `1:` after the comma
+You can also ask for multiple columns.
+We pass `1:` after the comma
 indicating we want columns after and including index 1 (*i.e.* `language`).
 
 ```{code-cell} ipython3
 tidy_lang.iloc[:, 1:]
 ```
 
-We can also use `iloc[]` to select ranges of rows, using a similar syntax.
-For example to select the ten first rows we could use the following:
-
-```{code-cell} ipython3
-tidy_lang.iloc[:10, :]
-```
-
-`pandas` also provides a shorthand for selecting ranges of rows by using `[]`:
+We can also use `iloc[]` to select ranges of rows, or simultaneously select ranges of rows and columns, using a similar syntax.
+For example, to select the first five rows and columns after and including index 1, we could use the following:
 
 ```{code-cell} ipython3
-tidy_lang[:10]
```

Comment:
I got rid of this shorthand example entirely. I think it will confuse students with the `[]` operator earlier. They certainly might (will eventually) see it in the wild, but I think the priority here is to make sure everyone is understanding what we teach and to keep things contained, and not necessarily to cover every possible thing they'll see beyond the class